### PR TITLE
Updates creating releases in PHP Workflow

### PR DESCRIPTION
**Pull request by shahriar-vdc**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -104,26 +104,22 @@ jobs:
...
       run: composer test
 
-    - name: Create Release
-      id: create_release
-      uses: actions/create-release@v1
-      if: startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Pre-release
+      uses: marvinpinto/action-automatic-releases@latest
```
Comment (asbjornu) [inline .github/workflows/php.yml:108]: Why the change of GitHub Action to use here?

File: auth.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "http-basic": {
+    "repo.magento.com": {
+      "username": "8d8b26b69cc64ebaa3687fde19f69a51",
+      "password": "7970da75e8347fb327100f9066735f7d"
+    },
+    "packages.trollweb.no": {
+      "username": "trollweb",
+      "password": "OcIIPtJRtJ714jb2hWxHHjtu27RAdlarjqLULv7"
```
Comment (asbjornu) [inline auth.json:9]: Committing these secrets seems like spectacularly bad idea?

**Review by asbjornu — CHANGES_REQUESTED** [review]
What purpose does this PR serve?

File: .github/workflows/php.yml (hunk 1 above)
Comment (shahriar-vdc) [reply .github/workflows/php.yml:108]: `actions/create-release` has some issues that tries to create a tag even though a tag already exists, which caused this build to fail: https://github.com/SwedbankPay/swedbank-pay-magento2-core/runs/1861672894?check_suite_focus=true

I noticed you also replaced the package in Checkout module: SwedbankPay/swedbank-pay-magento2-checkout@5a0f542

File: auth.json (hunk 2 above)
Comment (shahriar-vdc) [reply auth.json]: This is absolutely my fault. Somehow I overlooked it. It's great that you noticed it.

File: .github/workflows/php.yml (hunk 1 above)
Comment (asbjornu) [reply .github/workflows/php.yml:108]: It would be great if the description of the pull request could contain these sorts of motivations and explanations. 👍🏼

**Review by asbjornu — APPROVED** [review]
